error: 
    status - 500 
    message - SyntaxError: Unexpected token { in JSON at position 7
    at JSON.parse (<anonymous>)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-26T07:10:03.254Z"}
error: uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1372:16)
    at listenInCluster (node:net:1420:12)
    at Server.listen (node:net:1508:7)
    at Function.listen (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\application.js:635:24)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\/index.js:37:5)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._compile (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.newLoader [as .js] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\node\lib\_babel-node.js:176:21)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32) {"date":"Thu May 26 2022 14:27:58 GMT+0530 (India Standard Time)","error":{"address":"::","code":"EADDRINUSE","errno":-4091,"port":3000,"syscall":"listen"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":353981},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19218836,"external":20818065,"heapTotal":78721024,"heapUsed":49363616,"rss":107986944},"pid":404,"uid":null,"version":"v16.15.0"},"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1372:16)\n    at listenInCluster (node:net:1420:12)\n    at Server.listen (node:net:1508:7)\n    at Function.listen (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\application.js:635:24)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js:37:5)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Module._compile (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Object.newLoader [as .js] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js:176:21)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)","timestamp":"2022-05-26T08:57:58.856Z","trace":[{"column":16,"file":"node:net","function":"Server.setupListenHandle [as _listen2]","line":1372,"method":"setupListenHandle [as _listen2]","native":false},{"column":12,"file":"node:net","function":"listenInCluster","line":1420,"method":null,"native":false},{"column":7,"file":"node:net","function":"Server.listen","line":1508,"method":"listen","native":false},{"column":24,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\application.js","function":"Function.listen","line":635,"method":"listen","native":false},{"column":5,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js","function":null,"line":37,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":24,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":7,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":21,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js","function":null,"line":176,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false}]}
error: 
    status - 500 
    message - TypeError: Cannot read properties of null (reading 'firstName')
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:16:24)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-26T10:31:58.552Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:13:07 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360290},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18557768,"external":20131505,"heapTotal":46653440,"heapUsed":43374304,"rss":77197312},"pid":10360,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:43:07.763Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:14:04 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360346},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18596094,"external":20222882,"heapTotal":78635008,"heapUsed":43616328,"rss":107712512},"pid":8436,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:44:04.385Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:16:49 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360511},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19665560,"external":25741805,"heapTotal":79683584,"heapUsed":56678800,"rss":109764608},"pid":11196,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:46:49.307Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:17:39 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360561},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\build\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18919592,"external":20411455,"heapTotal":38715392,"heapUsed":27658672,"rss":62693376},"pid":8464,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:47:39.361Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:25:09 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":361012},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\build\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18903648,"external":20412426,"heapTotal":38715392,"heapUsed":27817128,"rss":62320640},"pid":5744,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:55:09.844Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:27:25 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":361147},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\build\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18920827,"external":20417307,"heapTotal":38715392,"heapUsed":27652520,"rss":62607360},"pid":5148,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:57:25.265Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 20:41:36 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":376399},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18473528,"external":20136923,"heapTotal":45604864,"heapUsed":41631864,"rss":75907072},"pid":3704,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T15:11:37.112Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
error: 
    status - 500 
    message - Error: Password not match
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:22:13)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-27T03:15:23.682Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:17:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-27T03:15:41.383Z"}
error: 
    status - 500 
    message - ReferenceError: userData is not defined
    at _callee2$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:10:43)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.registerUser (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:9:26)
    at _callee2$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/user.controller.js:25:36)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:25:14.458Z"}
error: 
    status - 500 
    message - Error: User already exist
    at _callee2$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:15:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:27:54.735Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected token , in JSON at position 77
    at JSON.parse (<anonymous>)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:33:52.540Z"}
error: 
    status - 500 
    message - ValidationError: "email" must be a valid email 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:38:40.762Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\validators\/user.validator.js:12:16)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:40:19.154Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\validators\/user.validator.js:12:16)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:40:24.374Z"}
error: 
    status - 500 
    message - Error: Password not match
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:33:13)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T05:27:55.333Z"}
error: 
    status - 500 
    message - Error: Password not match
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:30:13)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-28T04:11:53.306Z"}
error: 
    status - 500 
    message - Error: secretOrPrivateKey must have a value
    at Object.sign (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\jsonwebtoken\sign.js:107:20)
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:29:23)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T03:42:32.717Z"}
error: 
    status - 500 
    message - Error: secretOrPrivateKey must have a value
    at Object.sign (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\jsonwebtoken\sign.js:107:20)
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:29:23)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T03:43:30.593Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:25:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T04:41:20.150Z"}
error: uncaughtException: Router.use() requires a middleware function but got a Object
TypeError: Router.use() requires a middleware function but got a Object
    at Function.use (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:469:13)
    at routes (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\routes\/index.js:16:10)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\/index.js:32:32)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._compile (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.newLoader [as .js] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\node\lib\_babel-node.js:176:21)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {"date":"Mon May 30 2022 10:14:24 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":684366},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19041760,"external":20636006,"heapTotal":79712256,"heapUsed":54380144,"rss":108691456},"pid":3920,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Router.use() requires a middleware function but got a Object\n    at Function.use (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\router\\index.js:469:13)\n    at routes (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\routes\\/index.js:16:10)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js:32:32)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Module._compile (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Object.newLoader [as .js] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js:176:21)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)","timestamp":"2022-05-30T04:44:24.189Z","trace":[{"column":13,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\router\\index.js","function":"Function.use","line":469,"method":"use","native":false},{"column":10,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\routes\\/index.js","function":"routes","line":16,"method":null,"native":false},{"column":32,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js","function":null,"line":32,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":24,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":7,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":21,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js","function":null,"line":176,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false}]}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T06:49:53.619Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:14.374Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:25.262Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:38.160Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:44.699Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.createNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:24)
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:6:38)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T07:11:44.947Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.createNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:24)
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:6:38)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T08:09:27.494Z"}
error: 
    status - 500 
    message - ReferenceError: UserService is not defined
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:30:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at updateNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:30:24)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:365:14) 
    url - /api/v1/notes/62947bfc2d56650e080715d4 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-05-30T09:23:26.458Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:27:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.deleteUser (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:27:24)
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:45:25)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes/62947bfc2d56650e080715d4 
    method - DELETE 
    IP - ::1
   {"timestamp":"2022-05-30T09:26:00.612Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes/62956fdc075c272174a262fb 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T02:21:44.867Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes/62956fdc075c272174a262fb 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T02:22:01.096Z"}
error: 
    status - 500 
    message - ReferenceError: UserService is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:4:28)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at getAllNotes (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:4:25)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12) 
    url - /api/v1/notes/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T02:52:04.674Z"}
